src/chartkit/components/HorizontalBar.js

```diff
@@ -43,12 +44,16 @@ class HorizontalBar extends Component {
 
     if (data) {
       const { index, indexValue } = data;
-      this.setState({ highlightedIndex: index, highlightedIndexValue: indexValue });
+      const label = {
+        highlightedIndex: index,
+        highlightedIndexValue: indexValue
+      };
+      this.setState(label);
       if (this.props.analyticsTracking) {
         trackUserInteraction({
           category: this.props.analyticsTracking.category,
-          action: `Chart Bar: ${TRACKING_EVENTS.actions.hover}`,
-          label: `${data.indexValue}: ${data.id}`,
+          action: `Chart Bar : ${TRACKING_EVENTS.actions.hover}`,
+          ...(label && { label: isObject(label) ? JSON.stringify(label) : label }),
```

Comment:
Why did this change? seems like it now includes less information than before where it was tracking  for example`{"Pediatric Brain Tumors: CBTTC": "probands"}` which tells helps to distinguish which part of the stacked bar the user hovered

Reply:
Agreed. Reverted it back to:
```javascript
const { id, index, indexValue } = data;
this.setState({ highlightedIndex: index, highlightedIndexValue: indexValue });
if (this.props.analyticsTracking) {
  trackUserInteraction({
    category: this.props.analyticsTracking.category,
    action: `Chart Bar: ${TRACKING_EVENTS.actions.hover}`,
    label: `${indexValue}: ${id}`,
  });
}
```